internal/modules/overview/printer/pod_test.go

```diff
@@ -15,23 +15,53 @@ import (
 	"github.com/stretchr/testify/require"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 
 	"github.com/vmware/octant/internal/conversion"
 	"github.com/vmware/octant/internal/testutil"
+	"github.com/vmware/octant/pkg/plugin"
+	"github.com/vmware/octant/pkg/store"
 	"github.com/vmware/octant/pkg/view/component"
 )
 
+var (
```

Comment:
These globals in tests was a sloppy pattern I introduced. We should clean these up at every opportunity we get. This should be in a test function somewhere.

Reply:
Cleaned this up a bit more